### PR TITLE
Fix french

### PR DESCRIPTION
**Pull request by bertrandlalo**
Here is : 
- [minor/fr] style/typo correction, based on friends review
- [major/core] change the link from [this article](https://www.aftenposten.no/meninger/debatt/i/P9ALzX/selvrapporteringssystem-for-overvaaking-av-korona-maa-paa-plass-naa-petter-bae-brandtzaeg)  to [this one](https://medium.com/@tomaspueyo/coronavirus-act-today-or-people-will-die-f4d3d9cd99ca) instead, which is easy to read and in english, relates the same subject of difficulty and importance to estimate the number of _real_ cases.

### CODE REVIEW

**Comment by fossecode** [comment]
Is it supposed to be files for both `fr` and `fr-fr`?

**Comment by bertrandlalo** [comment]
No, it's me messing up with my branches. I removed the 'fr' versions.

File: app/views/privacy-statements/fr-fr-lang-privacy-statement.ejs [hunk 1]
```diff
@@ -69,7 +69,7 @@
 <h3>D'autres questions?</h3>
 <p>
   Contactez nous sur Twitter:
-  <a class="text-cvs-red hover:underline" href="https://twitter.com/coronastatusFR" target="_blank"
-    >@coronastatusFR</a
+  <a class="text-cvs-red hover:underline" href="https://twitter.com/coronastatusNL" target="_blank"
```
Comment (fossecode) [inline app/views/privacy-statements/fr-fr-lang-privacy-statement.ejs]: Woops, should probably be FR here

Comment (bertrandlalo) [reply app/views/privacy-statements/fr-fr-lang-privacy-statement.ejs]: super-👁, tks!

**Review by fossecode — APPROVED** [review]

**Pull request merged**